lib/plugin-compat.php

```diff
@@ -27,7 +27,12 @@
  */
 function gutenberg_remove_wpcom_markdown_support( $post ) {
 	if ( class_exists( 'WPCom_Markdown' ) && gutenberg_content_has_blocks( $post['post_content'] ) ) {
-		WPCom_Markdown::get_instance()->unload_markdown_for_posts();
+		if (
```

Comment:
Should this `if` be collapsed into its parent since it's effectively redundant nesting?

```php
// Before:
if ( $foo > 10 ) {
	if ( $foo > 100 ) {
		// ...
	}
}

// After:
if ( $foo > 10 && $foo > 100 ) {
	// ...
}
```

Reply:
I agree that they should be collapsed. Fixed.